File: Java/logica/src/Vetores.java
import java.util.Locale;
import java.util.Scanner;

public class Vetores {
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int N;
        System.out.print("Quantos quartos serão alugados? ");
        N = sc.nextInt();

        int[] vet = new int[N];

        for (int i = 0; i < N; i++) {
            System.out.print("Digite um número: ");
            vet[i] = sc.nextInt();
        }

        System.out.println("Números digitados: ");
        for (int i = 0; i < N; i++) {
            System.out.println(vet[i]);
        }

        sc.close();
    }
}

File: Python/Vetores.py
N = int(input("Quantos números você quer digitar? "))

vet: [float] = [0 for x in range(N)]

for i in range(0, N):
  vet[i] = float(input(f"digite um número: "))

print()
print("Números digitados: ")
for i in range(0, N):
  print(f"{vet[i]:.1f}")